master/buildbot/steps/transfer.py

```diff
@@ -383,15 +388,49 @@ def start(self):
             return self.finished(SKIPPED)
 
         @defer.inlineCallbacks
-        def uploadSources():
-            for source in sources:
-                result = yield self.startUpload(source, masterdest)
-                if result == FAILURE:
-                    defer.returnValue(FAILURE)
-                    return
-            defer.returnValue(SUCCESS)
+        def globSources(sources):
+            dl = defer.DeferredList([
+                self.runGlob(
```

Comment:
I am not sure we actually can run several commands on the same builderforworker at the same time. 
Might work, but we need an integration test for it